docs/content/docs/dev/datastream/user_defined_functions.md

```diff
@@ -129,8 +133,8 @@ you can write
 
 ```scala
 class MyMapFunction extends RichMapFunction[String, Int] {
-  def map(in: String):Int = { in.toInt }
-};
+  def map(in: String): Int = in.toInt
```

Comment:
inconsistent use of spaces after `:`

Reply:
Hi @zentol ,
Thank you for your reviewing this issue. I have made it consistent by adding one space after `:` .